apps/system/utils/utils_ps.c

```diff
@@ -134,11 +134,16 @@ static void ps_print_values(char *buf, void *arg)
 		return;
 	}
 #endif
-	
-	printf("%5s | %4s | %4s | %7s | %c%c | %8s | %3s", stat_info[PROC_STAT_PID], stat_info[PROC_STAT_PRIORITY], \
+
+	printf("%5s | %4s | %4s | %7s | %c%c | %8s", \
+		stat_info[PROC_STAT_PID], stat_info[PROC_STAT_PRIORITY], \
 		flags & TCB_FLAG_ROUND_ROBIN ? "RR  " : "FIFO", utils_ttypenames[(flags & TCB_FLAG_TTYPE_MASK) >> TCB_FLAG_TTYPE_SHIFT], \
 		flags & TCB_FLAG_NONCANCELABLE ? 'N' : ' ', flags & TCB_FLAG_CANCEL_PENDING ? 'P' : ' ', \
-		utils_statenames[state], stat_info[PROC_STAT_CPU]);
+		utils_statenames[state]);
+
+#ifdef CONFIG_SMP
```

Comment:
I suggest enabling all enum without any conditional and if some conditional is disabled, printing NA, 0 or something like this.